modules/claims_api/app/services/claims_api/slack/failed_submissions_messenger.rb

```diff
@@ -69,6 +75,12 @@ def build_submission_information(errored_submissions, submission_type)
         end
         errored_submission_message
       end
+
+      def link_value(id)
+        "<https://vagov.ddog-gov.com/logs?query='#{id}'&agg_m=count&agg_m_source=base&agg_t=count&cols=" \
+          'host%2Cservice&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=' \
+          "desc&viz=stream&from_ts=#{(Time.now.to_i - (3 * 24 * 60 * 60)).to_i}&to_ts=#{Time.now.to_i}&live=true|"
+      end
```

Comment:
The idea here is that when the alert is sent we set the window to be from there back 3 days.  This way if you come in Monday morning you can still view the Friday evening alerts without any change.  Since it is based on an ID value that window should not add any additional noise to the search